server/models/timeModel.js

```diff
@@ -0,0 +1,43 @@
+class Time{
+    constructor(day, hour, min, sec){
+        this.day = day;
+        this.dayNumber = this.convertDayIntoNumber(day);
+        this.hour = hour;
+        this.min = min;
+        this.sec = sec;
+    }
+
+    //METHODS
+    convertDayIntoNumber(day){
+        switch(day){
+          case "Monday":
+            day = "1";
+            break;
+          case "Tuesday":
+            day = "2";
+            break;
+          case "Wednesday":
+            day = "3";
+            break;
+          case "Thursday":
+            day = "4";
+            break;
+          case "Friday":
+            day = "5";
+            break;
+          case "Saturday":
+            day = "6";
+            break;
+          case "Sunday":
+            day = "7";
```

Comment:
This is a bug, Sunday has index 0.

Reply:
It's only a bug if I want to export to jdays. It's 7 instead of 0 because I made dates as comparable "numeric strings" ( <, >, =). Sunday comes after Saturday, so the string "70000" > "60000". But if we decided to use jdays, this whole file is useless and to be deleted.